practice/3_python_testing/test_task_parametrize.py

```diff
@@ -19,7 +19,17 @@ def fibonacci_1(n):
 
 
 def fibonacci_2(n):
-    fibo = [0, 1]
-    for i in range(1, n+1):
-        fibo.append(fibo[i-1] + fibo[i-2])
-    return fibo[n]
+    if n <= 0:
```

Comment:
I think it is not necessary to add here this if. we are not going to test the function with negative numbers. and other cases will work without the if